From 48b707feb937d2d03fd10a1f367cf08ed4c952a3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 28 Oct 2018 19:45:51 +0200
Subject: [PATCH] uboot: goya/gaudi: update fragments

The CONFIG_BOOTCOMMAND and CONFIG_SPI_FLASH_MACRONIX are the same
across habanalabs devices and have been introduced in the U-Boot
defconfig files.
Remove from fragments.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/gaudi/uboot_asic.fragment b/board/habanalabs/gaudi/uboot_asic.fragment
index 7fda35e..745c14e 100644
--- a/board/habanalabs/gaudi/uboot_asic.fragment
+++ b/board/habanalabs/gaudi/uboot_asic.fragment
@@ -3,9 +3,7 @@
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
 CONFIG_DEBUG_UART_CLOCK=107500000
-CONFIG_BOOTCOMMAND="while true; do waitcmd; run command; done"
 CONFIG_DEFAULT_DEVICE_TREE="gaudi_asic"
-CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_SYS_I2C_EEPROM_BUS=1
 CONFIG_SYS_EEPROM_SIZE=256
 CONFIG_SYS_EEPROM_PAGE_WRITE_BITS=4
diff --git a/board/habanalabs/gaudi/uboot_onix.fragment b/board/habanalabs/gaudi/uboot_onix.fragment
index 32d1571..1c27447 100644
--- a/board/habanalabs/gaudi/uboot_onix.fragment
+++ b/board/habanalabs/gaudi/uboot_onix.fragment
@@ -3,10 +3,8 @@
 CONFIG_BAUDRATE=57600
 CONFIG_BOOTDELAY=10
 CONFIG_DEBUG_UART_CLOCK=35000000
-CONFIG_BOOTCOMMAND="while true; do waitcmd; run command; done"
 CONFIG_DEFAULT_DEVICE_TREE="gaudi_onix"
 CONFIG_SPI_FLASH_STMICRO=y
-CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_SYS_I2C_EEPROM_BUS=1
 CONFIG_SYS_EEPROM_SIZE=256
 CONFIG_SYS_EEPROM_PAGE_WRITE_BITS=4
diff --git a/board/habanalabs/gaudi/uboot_pldm.fragment b/board/habanalabs/gaudi/uboot_pldm.fragment
index ac340d5..dc6f229 100644
--- a/board/habanalabs/gaudi/uboot_pldm.fragment
+++ b/board/habanalabs/gaudi/uboot_pldm.fragment
@@ -3,7 +3,5 @@
 CONFIG_BAUDRATE=1200
 CONFIG_BOOTDELAY=10
 CONFIG_DEBUG_UART_CLOCK=30400
-CONFIG_BOOTCOMMAND="while true; do waitcmd; run command; done"
 CONFIG_DEFAULT_DEVICE_TREE="gaudi_pldm"
-CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_PHY_FIXED=y
diff --git a/board/habanalabs/goya/uboot_asic.fragment b/board/habanalabs/goya/uboot_asic.fragment
index e5f5c4f..f92c510 100644
--- a/board/habanalabs/goya/uboot_asic.fragment
+++ b/board/habanalabs/goya/uboot_asic.fragment
@@ -3,9 +3,7 @@
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
 CONFIG_DEBUG_UART_CLOCK=23888888
-CONFIG_BOOTCOMMAND="while true; do waitcmd; run command; done"
 CONFIG_DEFAULT_DEVICE_TREE="goya_asic"
-CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_SYS_I2C_EEPROM_BUS=1
 CONFIG_SYS_EEPROM_SIZE=256
 CONFIG_SYS_EEPROM_PAGE_WRITE_BITS=4
diff --git a/board/habanalabs/goya/uboot_onix.fragment b/board/habanalabs/goya/uboot_onix.fragment
index 5b96d21..4efcd6c 100644
--- a/board/habanalabs/goya/uboot_onix.fragment
+++ b/board/habanalabs/goya/uboot_onix.fragment
@@ -3,10 +3,8 @@
 CONFIG_BAUDRATE=57600
 CONFIG_BOOTDELAY=10
 CONFIG_DEBUG_UART_CLOCK=50000000
-CONFIG_BOOTCOMMAND="while true; do waitcmd; run command; done"
 CONFIG_DEFAULT_DEVICE_TREE="goya_onix"
 CONFIG_SPI_FLASH_STMICRO=y
-CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_SYS_I2C_EEPROM_BUS=1
 CONFIG_SYS_EEPROM_SIZE=256
 CONFIG_SYS_EEPROM_PAGE_WRITE_BITS=4
diff --git a/board/habanalabs/goya/uboot_pldm.fragment b/board/habanalabs/goya/uboot_pldm.fragment
index 9e2d50c..b93705b 100644
--- a/board/habanalabs/goya/uboot_pldm.fragment
+++ b/board/habanalabs/goya/uboot_pldm.fragment
@@ -3,7 +3,5 @@
 CONFIG_BAUDRATE=1200
 CONFIG_BOOTDELAY=10
 CONFIG_DEBUG_UART_CLOCK=25000
-CONFIG_BOOTCOMMAND="while true; do waitcmd; run command; done"
 CONFIG_DEFAULT_DEVICE_TREE="goya_pldm"
-CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_PHY_FIXED=y
